### ISSUE TITLE
Play button to play back preview?

### ISSUE THREAD

**Issue by SolarLune** [leafo/gifine#21]
It'd be really nice to have a play button to play back the preview so you can watch it before saving it out.

**Comment by leafo** [comment]
I agree, in the meantime you can click on the slider and hold your right arrow key to simulate playback. (that's what I've been doing)

**Comment by SolarLune** [comment]
Ah! That's a good workaround / alternative. Thanks!
